web-services/model/src/main/java/datawave/webservice/query/model/ModelBean.java

```diff
@@ -110,7 +124,12 @@
     @GZIP
     @Interceptors(ResponseInterceptor.class)
     public ModelList listModelNames(@QueryParam("modelTableName") @DefaultValue(DEFAULT_MODEL_TABLE_NAME) String modelTableName) {
-        ModelList response = new ModelList();
+        
+        if (modelTableName == null) {
```

Comment:
This will never be true because of the `@DefaultValue(DEFAULT_MODEL_TABLE_NAME)` annotation on line 112/126. You have to remove that annotation for this approach to work, and do the same thing in every other method using the approach.